etc/kayobe/kolla/globals.yml

```diff
@@ -17,3 +17,38 @@ cinder_backend_ceph: "yes"
 # TODO: how to build these?
 #magnum_tag: pike2
```

Comment:
Procedure for these would be to:
1. Set `kolla_openstack_release: pike2`, and update `kolla_source_version: ` to point to the pike branch/tag.
2. Set these tags below, eg. `magnum_tag: pike2`
3. Build and push image, it will be tagged with `pike2`
4. Pull down the image across the overcloud and deploy the services via Kayobe. The `pike2` image tag will be used only for these services.